test/test_board.py

```diff
@@ -4,163 +4,85 @@
-        self.assertEqual(False, board.is_full(total_marks_on_board, full_board))
+        not_full_board = ["X", "2", "3", "4", "X", "O", "X", "O", "X"]
+        total_marks_on_board = self.board.count_marks(not_full_board, "X", "O")
+        self.assertFalse(self.board.is_full(total_marks_on_board, not_full_board))
 
     def test_is_full_returns_true_if_board_is_full_with_emojis(self):
-        board = Board()
         full_board = ["🤡", "👻", "🤡", "👻", "🤡", "👻", "🤡", "👻", "🤡"]
-        player_one = "🤡"
-        player_two = "👻"
-        total_marks_on_board = board.count_marks(full_board, player_one, player_two)
-        self.assertEqual(True, board.is_full(total_marks_on_board, full_board))
+        total_marks_on_board = self.board.count_marks(full_board, "🤡", "👻")
+        self.assertTrue(self.board.is_full(total_marks_on_board, full_board))
 
     def test_no_turns_taken_yet(self):
-        board = Board()
-        current_board = ["1", "2", "3", "4", "5", "6", "7", "8", "9"]
-        player_one = "X"
-        player_two = "O"
-        total_marks_on_board = board.count_marks(current_board, player_one, player_two)
+        total_marks_on_board = self.board.count_marks(self.starter_board, "X", "O")
         self.assertEqual(0, total_marks_on_board)
 
     def test_one_turn_taken_returns_one_mark_on_board(self):
-        board = Board()
         current_board = ["X", "2", "3", "4", "5", "6", "7", "8", "9"]
-        player_one = "X"
-        player_two = "O"
-        total_marks_on_board = board.count_marks(current_board, player_one, player_two)
+        total_marks_on_board = self.board.count_marks(current_board, "X", "O")
         self.assertEqual(1, total_marks_on_board)
 
     def test_two_turns_taken_returns_two_marks_on_board(self):
-        board = Board()
         current_board = ["X", "O", "3", "4", "5", "6", "7", "8", "9"]
-        player_one = "X"
-        player_two = "O"
-        total_marks_on_board = board.count_marks(current_board, player_one, player_two)
+        total_marks_on_board = self.board.count_marks(current_board, "X", "O")
         self.assertEqual(2, total_marks_on_board)
 
     def test_one_turn_taken_returns_one_mark_on_board_with_emoji(self):
-        board = Board()
         current_board = ["1", "🤡", "3", "4", "5", "6", "7", "8", "9"]
-        player_one = "🤡"
-        player_two = "👻"
-        total_marks_on_board = board.count_marks(current_board, player_one, player_two)
+        total_marks_on_board = self.board.count_marks(current_board, "🤡", "👻")
         self.assertEqual(1, total_marks_on_board)
 
-    def test_board_is_marked_with_user_selection_when_no_marks_and_X_is_going_first(
-        self,
-    ):
-        board = Board()
-        current_board = ["1", "2", "3", "4", "5", "6", "7", "8", "9"]
-        new_board_with_marks = ["X", "2", "3", "4", "5", "6", "7", "8", "9"]
-        user_input = 1
+    def test_mark_board_marks_board_with_user_selection(self):
```

Comment:
While this is good, I still suggest keeping the "arrange -> act -> assert" pattern in your tests. This helps to keep it human readable.  

```python
    def test_mark_board_marks_board_with_user_selection(self):
        test_mark_one = "X"
        test_mark_two = "O"
        test_mark_three = "🤡"
        expected_board = [test_mark_one, test_mark_two, test_mark_one, test_mark_three, "5", "6", "7", "8", "9"]

        self.board.mark_board(1, self.starter_board, test_mark_one)
        self.board.mark_board(2, self.starter_board, test_mark_two)
        self.board.mark_board(3, self.starter_board, test_mark_one)
        self.board.mark_board(4, self.starter_board, test_mark_three)

        self.assertEqual(self.board, expected_board)
```